CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+#
```

Comment:
Since our branch protection rules works on the configured contributors with write access, this file is not required.